### PR TITLE
Fix/explicit attributes true: just measures in active attrs and active attrs with expressions

### PR DESCRIPTION
**Pull request by AlvaroVega**
- [x] Add more tests about explicitAttributes: true

### CODE REVIEW

**Review by mapedraza — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+- Fix: explicitAttributes true case: should progress just active attributes which receives measures and all active attributes with expressions
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: This behaviour match the one described in documentation?

Comment (mapedraza) [reply CHANGES_NEXT_RELEASE:1]: According to this:

https://github.com/telefonicaid/iotagent-node-lib/blob/ea48c800eaca591441a70be56b34d61b93b93dc9/doc/advanced-topics.md?plain=1#L301-L309

> *the IoTAgent rejects the  measure elements that are not defined in the mappings of device or group provision*

From my perspective, yes, it impact on it

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/advanced-topics.md#explicitly-defined-attributes-explicitattrs

Case 2:

"explicitAttrs": true

just measures defined in active, static (plus conditionally TimeInstant) will be propagated to NGSI interface.

Comment (mapedraza) [reply CHANGES_NEXT_RELEASE:1]: So, no further changes required in the documentation

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**